contrib/core-contract-tests/tests/sip-031/sip-031.test.ts

```diff
@@ -0,0 +1,320 @@
+import { project, accounts } from '../clarigen-types'; // where your [types.output] was specified
```

Comment:
@hstove do you run those tests in `contrib/core-contract-tests` with `npx vitest tests/sip-031/sip-031.test.ts`?